srcCpp/PerftestTransport.cxx

```diff
@@ -1115,4 +1132,53 @@ const std::string PerftestTransport::getMulticastAddr(const char *topicName)
     return address;
 }
 
+bool PerftestTransport::parseCustomMulticastAddresses(char *arg)
```

Comment:
I like this approach because it will be very easy to integrate it with the parameter manager!